dev/config.yaml

```diff
@@ -5,3 +5,7 @@ database:
     connections:
       - host: 0.0.0.0
         port: 2379
+
+schemas:
+  path: "schemas"
+  interval: "1h"
```

Comment:
github doesn't seem to like when there is not line at the end of the file.
Probably we should enforce it at some point